content/en/docs/features/control-plane/entity-tag.md

```diff
@@ -114,7 +78,7 @@ There are two methods for updating the ETAG when a resource changes:
 
 ### ETAG Usage by Clients
 
-In order to improve network traffic efficiency, clients have the option to store received ETAGs from GET responses in a cache. To determine if the ETAG is still valid, the client can send the ETAG in a request to the device. If the ETAG is valid, the device will respond with a `VALID` code and no body. If the ETAG is not valid, the device will respond with a `CONTENT` code containing the current ETAG and the corresponding body.
+In order to improve network traffic efficiency, clients have the option to store received ETAGs from GET responses in a cache. To determine if the ETAG is still valid, the client can send the ETAGs in a request to the device. If the ETAG is valid, the device will respond with a `VALID` code and no body, and corresponding matched ETAG. If the ETAG is not valid, the device will respond with a `CONTENT` code containing the current ETAG and the corresponding body.
```

Comment:
So ETag should be send also for 'VALID' code and not only for 'CONTENT'? That's for the use case if you send multiple ETags to know which matched or something else?

Reply:
Yes - For VALID(to determine which etag is valid) and CONTENT.